tests/matchers/test_callable.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+from robber import expect
+from robber.matchers.callable import Callable
+
+
+class TestCallable(TestCase):
+    def test_matches(self):
+        def a():
+            pass
+
+        expect(Callable(a).matches()) == True
+
+    def test_failure_message(self):
+        callable = Callable("a")
```

Comment:
And according to my above explaination, `callable` is a builtin function, so it's better that we choose another name.